### PR TITLE
Create Parms class that contain information from client

### PR DESCRIPTION
**Pull request by luqp**

### CODE REVIEW

File: src/main/java/org/jalasoft/moi/model/Params.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package org.jalasoft.moi.model;
+
+import java.nio.file.Path;
+
+/**
+ * Parms class contains information from client about what language and root are need
```
Comment (nanrehd) [inline src/main/java/org/jalasoft/moi/model/Params.java]: Please check the description suits the class name

**Review by nanrehd — DISMISSED** [review]

File: src/main/java/org/jalasoft/moi/model/Params.java (hunk 1 above)
Comment (luqp) [reply src/main/java/org/jalasoft/moi/model/Params.java]: ok

File: src/main/java/org/jalasoft/moi/model/Laguage.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * Copyright (c) 2020 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package org.jalasoft.moi.model;
+
```
Comment (carlosmc23) [inline src/main/java/org/jalasoft/moi/model/Laguage.java:11]: I think you need comment this enum and explain what is for

File: src/main/java/org/jalasoft/moi/model/Params.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+     * @param filesPath contains the root files in the local host.
+     * @param version contains the language version.
+     * @param language of the code used for the client.
+     */
+    public Params(Path filesPath, String version, Laguage language) {
+        this.filesPath = filesPath;
+        this.version = version;
+        this.language = language;
+    }
+
```
Comment (carlosmc23) [inline src/main/java/org/jalasoft/moi/model/Params.java:32]: please add the setters method,  I think we will need setters

File: src/main/java/org/jalasoft/moi/model/Laguage.java (hunk 2 above)
Comment (luqp) [reply src/main/java/org/jalasoft/moi/model/Laguage.java:11]: I was think so, but I'm not sure what should put in it, Could you give me some ideas?

**Pull request closed without merging**